components/code-editor/index.js

```diff
@@ -76,6 +77,11 @@ class LazyCodeEditor extends Component {
 		this.state = {
 			status: 'pending',
 		};
+
+		deprecated( 'wp.components.CodeEditor', {
+			version: '4.2',
+			alternative: 'wp.codeEditor directly or any alternative React component for syntax highlighting',
```

Comment:
This is going to be confusing in the context of npm package which is out of WordPress context. Not sure if this is bad or not, just noting.

Reply:
yes, we have a bunch of those and we need to experiment with making the messages opt-in or something and only enable in WP